notebooks/image_models/solutions/diffusion_model.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<p>Consider adding a markdown cell above explaining a few of these constants, especially: <code>CENTRAL_CROP</code>,  <code>MIN_SIGNAL_RATE</code>,  <code>MAX_SIGNAL_RATE</code>,  <code>PLOT_DIFFUSION_STEPS</code> , and <code>EMA</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Done.</p>